sapl/relatorios/views.py

```diff
@@ -1880,6 +1885,75 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class RelatorioVotacoesNominaisView(RelatorioMixin, VotacoesMultiFormatOutputMixin, FilterView):
+    model = VotoParlamentar
+    filterset_class = RelatorioVotacoesNominaisFilterSet
+    template_name = 'relatorios/RelatorioVotacoesNominais_filter.html'
+    relatorio = relatorio_votacao_nominal
+
+    fields_base_report = [
+        'votacao_id', 'votacao', 'parlamentar__nome_parlamentar', 'voto'
+    ]
+
+    fields_report = {
+        'csv': fields_base_report,
+        'xlsx': fields_base_report,
+        'json': fields_base_report,
+    }
+
+    def get_context_data(self, **kwargs):
+        context = super(
+            RelatorioVotacoesNominaisView, self
+        ).get_context_data(**kwargs)
+
+        context['title'] = _('Votações Nominais')
+
+        if not self.filterset.form.is_valid():
+            return context
+
+        query_dict = self.request.GET.copy()
+        context['filter_url'] = ('&' + query_dict.urlencode()) if len(query_dict) > 0 else ''
+        context['show_results'] = show_results_filter_set(query_dict)
+
+        data_inicial = self.request.GET['data_hora_0']
+        data_final = self.request.GET['data_hora_1']
```

Comment:
Esse tipo de construção vai gerar `KeyError` se a chave não existir. Ao invés disso, use `.get()` com default value:

```suggestion
        data_inicial = self.request.GET.get('data_hora_0', '')
        data_final = self.request.GET.get('data_hora_1', '')
```